test/hpr_http_roaming_packet_SUITE.erl

```diff
@@ -0,0 +1,1588 @@
+%%%-------------------------------------------------------------------
+%%% @author jonathanruttenberg
+%%% @copyright (C) 2022, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 06. Oct 2022 2:11 PM
+%%%-------------------------------------------------------------------
+-module(hpr_http_roaming_packet_SUITE).
+-author("jonathanruttenberg").
+
+-define(TRANSACTION_ETS, hpr_http_roaming_transaction_ets).
+
+-export([
+    all/0,
+    init_per_testcase/2,
+    end_per_testcase/2
+]).
+
+-export([
+    http_sync_uplink_join_test/1,
+    http_sync_downlink_test/1,
+    http_async_uplink_join_test/1,
+    http_async_downlink_test/1,
+    http_uplink_packet_no_roaming_agreement_test/1,
+    http_uplink_packet_test/1,
+    http_class_c_downlink_test/1,
+    http_multiple_gateways_test/1,
+    http_multiple_joins_same_dest_test/1,
+    http_multiple_gateways_single_shot_test/1,
+    http_overlapping_devaddr_test/1,
+    http_uplink_packet_late_test/1
+]).
+
+%% Elli callback functions
+-export([
+    handle/2,
+    handle_event/3
+]).
+
+-export([
+    http_rcv/1
```

Comment:
Is this being used anywhere else?

Reply:
No. I will remove the export.

Reply:
@michaeldjeffrey 
In fact, without the export, a test fails because it calls ```?MODULE:http_rcv```. Calling in this fashion, the export is required.
If there is no value in calling this way, then I can not mention the MODULE and remove the export.
What do you recommend?

Reply:
...
Removed ?MODULE call and export.